### PR TITLE
Add mssql_tls_remote_src

### PR DESCRIPTION
**Pull request by spetrosi**
SQL Server role should be able to use TLS cert and key files that are
already on managed nodes

### CODE REVIEW

**Comment by spetrosi** [comment]
[citest pending]

**Comment by spetrosi** [comment]
[citest pending]

File: tests/tests_tls_2019.yml [hunk 1]
```diff
@@ -80,5 +81,39 @@
...
+    - name: Remove certificates from hosts
+      file:
+        path: /etc/pki/tls/{{ item }}
+        state: absent
+      loop: "{{ __mssql_certs }}"
+
+    - name: Copy certificates to hosts
+      copy:
+        src: "{{ item }}"
+        dest: ~/{{ item }}
```
Comment (richm) [inline tests/tests_tls_2019.yml:94]: using `~` in this context works?

File: tasks/main.yml [hunk 2]
```diff
@@ -420,6 +420,7 @@
     - name: Copy certificate and private_key files to the host
       copy:
         src: "{{ item }}"
+        remote_src: "{{ mssql_tls_remote_src }}"
```
Comment (richm) [inline tasks/main.yml:423]: So this assumes the cert and key are already on the remote host, just not in the right location?

Comment (spetrosi) [reply tasks/main.yml:423]: Yes, e.g. when using the certificate role to generate certificates and then must move them to the location required for mssql

File: tests/tests_tls_2019.yml (hunk 1 above)
Comment (spetrosi) [reply tests/tests_tls_2019.yml:94]: It did when I run CI locally with tox

**Comment by richm** [comment]
[citest]

File: tests/tests_tls_2019.yml (hunk 1 above)
Comment (spetrosi) [reply tests/tests_tls_2019.yml:94]: I should rather copy them to /tmp/{{ item }} though

**Comment by spetrosi** [comment]
[citest pending]

**Comment by spetrosi** [comment]
[citest]

**Review by richm — APPROVED** [review]
lgtm

**Pull request merged**